scramv1-tool-conf.file

```diff
@@ -8,18 +8,22 @@ Source: none
 %define CheckScramTools \
   uctool=`echo $tool | tr '[a-z-]' '[A-Z_]'` \
   toolbase=`eval echo \\\\$${uctool}_ROOT` \
-  [ -d $toolbase/etc/scram.d ] || continue
+  [ -d ${SCRAM_TOOLS_ROOT}/scram-tools/$tool ] || continue
 
 %define CopyScramTools \
   echo ">> Copying tool files from: $tool" \
-  for xml in $(find $toolbase/etc/scram.d -type f) ; do \
+  for xml in $(find ${SCRAM_TOOLS_ROOT}/scram-tools/$tool -type f -name "*.xml") ; do \
     bxml=$(basename $xml | tr '[A-Z]' '[a-z]') \
     [ -f %i/tools/selected/$bxml ] && continue \
     [ -f %i/tools/available/$bxml ] && continue \
     cp $xml %i/tools/selected/${bxml} \
+    T_VERSION=`eval echo \\\\$${uctool}_VERSION` \
+    sed -i -e 's|@TOOL_ROOT@|'"$toolbase"'|g' %i/tools/selected/${bxml} \
+    sed -i -e 's|@TOOL_VERSION@|'"$T_VERSION"'|g' %i/tools/selected/${bxml} \
     echo "  Copied $bxml" \
-  done
-
+  done \
+  tool_script=${SCRAM_TOOLS_ROOT}/scram-tools/$tool/$(echo $tool | tr '[A-Z]' '[a-z]').sh \
+  if [ -f $tool_script ]; then ($tool_script %i/tools/selected); fi
```

Comment:
@mrodozov , I was thinking to move all this `CopyScramTools` logic in a script which we can call like `scram-tools/get_tools <tool_name> <outdir>`. This tool then can process the `<toolname>/<toolname>*.xml` files and replace `@TOOL_ROOT@ and @TOOL_VERSION@` and can internally call `scram-tools/get_tools/<toolname>/<toolname>.sh` if available

Reply:
I'll move that part in a file to be called as a script with args then

Reply:
the `${SCRAM_TOOLS_ROOT}/get_tools` script can take care of common variables e.g. 
```
TOOL_ROOT
TOOL_VERSION
PYTHONV
PYTHON3V
```

Reply:
@mrodozov , please take note of my comment https://github.com/cms-sw/cmsdist/pull/7110#discussion_r665327856 too